src/StudyViewer/StudyDetails.jsx

```diff
@@ -138,17 +138,35 @@ class StudyDetails extends React.Component {
       && this.isDataAccessible(this.props.data.accessibleValidationValue)
       && this.props.fileData.length > 0;
 
-      button = displayDownloadButton ? (
-        <Button
-          key={key}
-          label={'Download'}
-          buttonType='primary'
-          onClick={this.showDownloadModal}
-          enabled={enableButton}
-          tooltipEnabled={tooltipEnabled}
-          tooltipText={tooltipText}
-        />
-      ) : null;
+      if (this.props.data.overrideDownloadUrlField && displayDownloadButton) {
+        button = (
+          <Button
+            key={key}
+            label={`Download ${this.props.data.requiredIdpField
+              ? `on ${this.props.data.requiredIdpField.toUpperCase()}`
+              : 'Externally'
+            }`}
```

Comment:
Even if this is how the NIAID use case works, i don't think we can assume that the download URL is always going to the "required IDP" website, or that it's always external to Gen3. Maybe the button text can still be "Download"? but the `rightIcon` is a nice touch